src/IFrameManager.js

```diff
@@ -98,22 +137,21 @@ class IFrameManager {
     element.innerHTML = "";
     element.appendChild(iFrame);
 
-    return element.querySelector("iFrame");
-  }
-
-  handleLoadError() {
-    this.onMessageHandler({
-      type: messageType.SYNC_ACK_FAILED,
-    });
+    return element.querySelector("iframe");
   }
 
+  /**
+   * @param {import('./typedefs').SizeInfo} sizeInfo
+   */
   updateIFrameSize(sizeInfo) {
     const { width, height } = sizeInfo;
-    const formatSize = size => (typeof size === "number" ? `${size}px` : size);
-    if (width) {
+    const formatSize = (/** @type {number} */ size) => {
+      return typeof size === "number" ? `${size}px` : size;
+    };
+    if (width && this.iFrame) {
       this.iFrame.setAttribute("width", formatSize(width));
     }
-    if (height) {
+    if (height && this.iFrame) {
```

Comment:
You could add a check for `this.iFrame` at the beggining of the function instead since everything in the function relies on it.

Reply:
done